shell/platform/windows/window_binding_handler.h

```diff
@@ -89,6 +89,9 @@ class WindowBindingHandler {
   // Returns the last known position of the primary pointer in window
   // coordinates.
   virtual PointerLocation GetPrimaryPointerLocation() = 0;
+
+  // Called to set the initial state of accessibility features
+  virtual void UpdateInitialAccessibilityFeatures() = 0;
```

Comment:
"Update" seems a bit strange here as we're setting initial values, what do you think of making this `SendInitialAccessibilityFeatures` as well?

```suggestion
  virtual void SendInitialAccessibilityFeatures() = 0;
```